### PR TITLE
USWDS - Blog - Accelerator post #3 - "Working with the U.S. Web Design System team"

### PR DESCRIPTION
**Pull request by sknep**
**Draft - waiting for social share images**

Adds new blog post for USWDS blog
## Working with the U.S. Web Design System team

Preview this blog post in context here: https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/accelerator/blog-post-3/whats-new/updates/2021/03/24/working-with-uswds-team/

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [x] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [x] Run `npm test` and make sure the tests for the files you have changed have passed.
- [x] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

**Comment by jonadecker** [comment]
@sknep [Tense mismatch](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/accelerator/blog-post-3/whats-new/updates/2021/03/04/working-with-uswds-team/#2-ask-for-feedback-early-and-often:~:text=reflecting%20that%20back%20into%20our%20work). Should be reflected. I've pinged @afeijoo on the source document in case she wants to overrule.

File: _posts/2021-03-04-working-with-uswds-team.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+This approach allowed UX and engineering pairs to work on specific components in parallel to get more done, mobbing with additional team members when tackling thorny issues while preserving the heads-down time needed for high quality work.
+
+### 2. Ask for feedback early and often
+
+Understanding the impact our decisions have on end-users of the design system as well as public users of the websites built using USWDS was critical to keeping us on the right path.
+
+We [interviewed digital teams across the federal government]({{ site.baseurl }}/whats-new/updates/2021/02/26/improving-emergency-response/), and got feedback from the product team, peers, and the [USWDS community]({{ site.baseurl }}/about/community/) to ensure we benefited from a diverse set of opinions, backgrounds, and experiences and reflecting that back into our work.
```
Comment (sknep) [inline _posts/2021-03-04-working-with-uswds-team.md:28]: Before publishing, please update this URL with the path to accelerator blog post #1

**Review by afeijoo — APPROVED** [review]
Update URL date from 3/4 to date published.

**Review by mejiaj — APPROVED** [review]

**Comment by afeijoo** [comment]
LGTM :shipit:

**Pull request merged**